staging/src/k8s.io/apimachinery/pkg/util/net/interface.go

```diff
@@ -84,9 +88,8 @@ func parseIP(str string) (net.IP, error) {
 	if err != nil {
 		return nil, err
 	}
-	//TODO add ipv6 support
 	if len(bytes) != net.IPv4len {
-		return nil, fmt.Errorf("only IPv4 is supported")
+		return nil, fmt.Errorf("invalid IPv4 address in route")
 	}
 	bytes[0], bytes[1], bytes[2], bytes[3] = bytes[3], bytes[2], bytes[1], bytes[0]
```

Comment:
yeah -- looks like there is no `ntohl` in golang. It might be less error prone to do
```
return net.IP([]byte{bytes[3], bytes[2], bytes[1], bytes[0]}), nil
```
and avoid the assignment altogether.